cfme/tests/containers/test_blacklisted_container_events.py

```diff
@@ -0,0 +1,208 @@
+        "image": "openshift/hello-openshift",
+        "ports": [
+          {
+            "containerPort": 8080,
+            "protocol": "TCP"
+          }
+        ],
+        "resources": {},
+        "volumeMounts": [
+          {
+            "name": "tmp",
+            "mountPath": "/tmp"
+          }
+        ],
+        "terminationMessagePath": "/dev/termination-log",
+        "imagePullPolicy": "IfNotPresent",
+        "capabilities": {},
+        "securityContext": {
+          "capabilities": {},
+          "privileged": False
+        }
+      }
+    ],
+    "volumes": [
+      {
+        "name":"tmp",
+        "emptyDir": {}
+      }
+    ],
+    "restartPolicy": "Always",
+    "dnsPolicy": "ClusterFirst",
+    "serviceAccount": ""
+  },
+  "status": {}
+}
+
+
+def create_pod(provider, namespace):
+    """Creates OpenShift pod in provided namespace"""
+
+    # TODO This method should be in wrapanapi, move when coverting to wrapanapi 3.0 for containers
+    provider.mgmt.k_api.create_namespaced_pod(namespace=namespace, body=TEST_POD)
+
+    pods = provider.mgmt.list_pods(namespace=namespace)
+
+    assert TEST_POD['metadata']['name'] == pods[0].metadata.name
+
+
+def delete_pod(provider, namespace):
+    """Delete OpenShift pod in provided namespace"""
+
+    provider.mgmt.delete_pod(namespace=namespace, name=TEST_POD['metadata']['name'])
+
+    wait_for(
+        lambda: not provider.mgmt.list_pods(namespace=namespace),
+        delay=5, num_sec=300,
+        message="waiting for pod to be deleted"
+    )
+
+    assert not provider.mgmt.list_pods(namespace=namespace)
+
+
+def appliance_cleanup(config, provider, appliance, namespace):
+    """Returns the appliance and provider to the original state"""
+
+    appliance.update_advanced_settings(config)
+    appliance.ssh_client.run_rails_console(
+        "BlacklistedEvent.where(:event_name => 'POD_CREATED').destroy_all")
+    appliance.evmserverd.restart()
+    appliance.wait_for_web_ui()
+
+    try:
+        delete_pod(provider=provider, namespace=namespace)
+        provider.mgmt.delete_project(name=namespace)
+
+    except ApiException:
+        logger.info("No Container Pod or Project to delete")
+
+
+def get_blacklisted_event_names(appliance):
+    """Returns a list of Blacklisted event names"""
+    rails_result = appliance.ssh_client.run_rails_console(
+        'ManageIQ::Providers::Openshift::ContainerManager.first.blacklisted_event_names')
+
+    assert rails_result.success
+
+    return rails_result.output
+
+
+def test_blacklisted_container_events(request, appliance, provider, app_creds):
+    """
+        Test that verifies that container events can be blacklisted.
+
+        Polarion:
+            assignee: juwatts
+            caseimportance: medium
+            initialEstimate: 1/6h
+    """
+    config = appliance.advanced_settings
+    original_conifg = deepcopy(config)
+
+    project_name = project_name = fauxfactory.gen_alpha(8).lower()
+
```

Comment:
@jawatts 
I suspect declaration of variable  ```project_name``` because same variable is assigned to same.

Reply:
Good catch, that was a copy paste error.